Neos.Utility.Lock/Classes/TYPO3/Flow/Utility/Lock/Lock.php

```diff
@@ -78,6 +77,18 @@ public static function setLockManager(LockManager $lockManager = null)
 
     /**
      * Releases the lock
+     * @param boolean $exclusiveLock TRUE to, acquire an exclusive (write) lock, FALSE for a shared (read) lock. An exclusive lock is the default.
+     * @return void
+     */
+    public function acquire($exclusiveLock)
```

Comment:
That's a bit irritating IMO - the constructor already takes the argument for exclusiveLock, but it must also be provided in acquire again?

Also, since acquire is now a public method, what is the expected behaviour if someone does this:
`$lock = new Lock(...); $lock->acquire(..);`?